src/blockly/generators/propc/sensors.js

```diff
@@ -1050,30 +1051,58 @@ Blockly.Blocks.lis3dh_init = {
                     .appendField(new Blockly.FieldNumber('0', 0, 100, 1), "SMOOTHING");
             this.setFieldValue(smoothing || '0', 'SMOOTHING');
 
+            // This has to appear above the voltage block if one is defined.
+            if (this.getInput('VOLT_CALIBRATE') ) {
+                this.moveInputBefore('TILT_CALIBRATE', 'VOLT_CALIBRATE');
+            }
         } else if (!hasTiltBlocks && this.getInput('TILT_CALIBRATE')) {
             this.removeInput('TILT_CALIBRATE');
             if (!this.getField('SMOOTHING')) {
                 this.getInput('PINS').appendField('', 'SMOOTHING');
             }
         }
     },
+    buildVoltageInput: function(hasVoltageBlocks) {
+        // Create a init field for the ADC voltage offset
+        if (hasVoltageBlocks && !this.getInput('VOLT_CALIBRATE')) {
+            // Save current value and then destroy the input field
+            var voltageField = this.getFieldValue('VOLTAGE');
+            this.getInput('PINS').removeField('VOLTAGE');
+
+            this.appendDummyInput('VOLT_CALIBRATE')
+                .appendField('Set the ADC voltage offset (+- 500 millivolts)')
```

Comment:
Just for consistency - we've used `+/-` on block text like this everywhere else.

Reply:
the word `voltage` is a bit redundant - since this is an ADC, you know you are measuring voltage, plus the units are `mV`, again, indicating voltage.  Fewer words total mean the block doesn't take up real-estate on the canvas.

Reply:
Updated the prompt per recommendation. Thanks!